objects/objects.shacl.ttl

```diff
@@ -694,6 +713,42 @@
       sh:message "premis:originalName heeft meer dan één waarde of is niet van het type string"@nl ;
       sh:message "premis:originalName a plus d'une valeur ou n'est pas de type string"@fr ;
       sh:severity sh:Violation ;
+    ] ,
+    [
+      sh:path  premis:identifier ;
+      sh:class mh:MediaObjectIdentifier ;
+
+      sh:minCount 1 ;
+      sh:maxCount 1 ;
+
+      sh:name "has media object identifier"@en ;
+      sh:name "heeft mediaobject-ID"@nl ;
+      sh:name "a identifiant d'objet multimédia"@fr ;
+
+      sh:description "De unieke mediaobject-ID van een object binnen het Mediahaven asset management system."@nl;
+      sh:description "The unique media object identifier of an object within the Mediahaven asset management system."@en;
+      sh:description "L'identifiant d'objet multimédia unique d'un objet au sein du système de gestion d'actifs Mediahaven."@fr;
+
+      sh:message "The object of premis:identifier is not a mh:MediaObjectIdentifier, is absent or occurs more than once."@en ;
+      sh:severity sh:Violation ;
+    ] ,
+    [
+      sh:path  premis:identifier ;
+      sh:class mh:RecordIdentifier ;
```

Comment:
Ik denk dat we deze eigenlijk niet nodig hebben, maar mag van mij in het model

Reply:
Je weet nooit...